trajectory_tt_publisher:
  ros__parameters:
    publishRate: 100.0
    publishOnce: true
    commandFrameId: command
    horizon: 5.0
    dt: 0.05
    amplitude: 0.20
    frequency: 0.5

    axisX: 1.0
    axisY: 0.0
    axisZ: 1.0

trajectory_progress_monitor:
  ros__parameters:
    monitorRate: 100.0
    publishTF: true
    closestFrameId: command_closest
    window: 20
    maxBacktrack: 5

    onTrackPos: 0.005
    onTrackOriDeg: 5.0

    divergedPos: 0.03
    divergedHoldSec: 0.5

    finishProgress: 0.98
    finishPos: 0.005
